lib/install/management/create.go

```diff
@@ -109,35 +117,97 @@ func (d *Dispatcher) startAppliance(conf *config.VirtualContainerHostConfigSpec)
 func (d *Dispatcher) uploadImages(files map[string]string) error {
 	defer trace.End(trace.Begin(""))
 
-	var wg sync.WaitGroup
-
 	// upload the images
 	log.Infof("Uploading images for container")
 
-	wg.Add(len(files))
-	results := make(chan error, len(files))
+	results := make(chan error)
```

Comment:
Don't understand why you need this either. All you're doing with the errors is printing them to the log. Why not just print them to the log? To know when you're done you could just use a `WaitGroup` as the old code was using.